components/alibi-detect-server/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # TODO: Add to release script 
-FROM docker.io/seldonio/seldon-core-s2i-python37-ubi8:1.3.0-dev
+FROM docker.io/seldonio/seldon-core-s2i-python37-ubi8:1.4.0-dev
```

Comment:
Shoud it not be 1.5.0-dev

Reply:
Oh wow good catch, yeah it should, I think i need to rebase actually... it seems this one is not covered so I shouldn't get any conflicts

Reply:
Ok this is now done 👍